@Khan1988 in `bayes_people_tracker_logging` (?)
```13:50:06 CMake Error at /opt/ros/kinetic/share/genmsg/cmake/genmsg-extras.cmake:263 (message):
13:50:06   Messages depends on unknown pkg: people_msgs (Missing
13:50:06   'find_package(people_msgs)'?)
13:50:06 Call Stack (most recent call first):
13:50:06   CMakeLists.txt:29 (generate_messages)```

--- a/bayes_people_tracker_logging/CMakeLists.txt
+++ b/bayes_people_tracker_logging/CMakeLists.txt
@@ -14,6 +14,7 @@ find_package(catkin REQUIRED COMPONENTS
   mdl_people_tracker
   tf
   upper_body_detector
+  people_msgs
 )
 
 ################################################
@@ -33,6 +34,7 @@ generate_messages(
     upper_body_detector
     std_msgs
     mdl_people_tracker
+    people_msgs
 )
 
 ###################################

